src/interface/report/Results/Timeline/Lane.js

```diff
@@ -110,9 +110,9 @@ class Lane extends React.PureComponent {
     const { children, style } = this.props;
 
     const ability = children[0].ability;
-    if(children[0].type === PRE_FILTER_COOLDOWN_EVENT_TYPE || children[0].type === "cast"){ //if first cast happened before phase
+    if(children[0].type === EventType.FilterCooldownInfo || children[0].type === "cast"){ //if first cast happened before phase
       const nextChildren = children.slice(1, children.length); //all children following the first cast
-      const nextCast = nextChildren.findIndex(e => e.type === "cast" || e.type === PRE_FILTER_COOLDOWN_EVENT_TYPE) + 1; //add 1 since we're searching through the events FOLLOWING the initial cast
+      const nextCast = nextChildren.findIndex(e => e.type === "cast" || e.type === EventType.FilterCooldownInfo) + 1; //add 1 since we're searching through the events FOLLOWING the initial cast
```

Comment:
```suggestion
      const nextCast = nextChildren.findIndex(e => e.type === EventType.Cast || e.type === EventType.FilterCooldownInfo) + 1; //add 1 since we're searching through the events FOLLOWING the initial cast
```